.github/workflows/release.yml

```
@@ -0,0 +1,51 @@
+name: release
```

Please capitalize and add a trailing newline for consistency.

```suggestion
name: Release

```

---

Done!